.github/workflows/ci.yml

```diff
@@ -74,7 +81,8 @@ jobs:
     runs-on: windows-2022
     strategy:
       matrix:
-        python-version: ["3.7", "3.10"]
+        python-version: ["3.7", "3.9", "3.10"]
```

Comment:
Why are we including 3.9 here and removing it from package build?

Reply:
I'm including it due to some differences in the importlib module. It's not relevant now but I think it will be to test against with some PR I've yet to submit.

Reply:
Let's leave that to the other PR then